cylc/flow/network/schema.py

```diff
@@ -633,7 +632,6 @@ class Meta:
         lambda: TaskProxy,
         description="""Task cycle instances.""",
         args=PROXY_ARGS,
-        ghosts=GHOSTS_DEFAULT,
```

Comment:
I don't really understand what "ghosts" were, why we had them or why we can dispense of them now?

Reply:
Ghosts where part of a crude n-window  (n=1)  prototype... It has long been deprecated, and need to be removed..
It relates do this PR because it was in the part of the filter I had to change.

Reply:
Ok, so I can test this is still working correctly just be running the UI on a workflow and making sure the N=1 window shows as expected.

Reply:
(Note the original movtivation for this PR was https://github.com/cylc/cylc-uiserver/issues/341 - i.e., custom task outputs not showing up at the UI in real time... which this PR definitely fixes.)

Reply:
> Ok, so I can test this is still working correctly just be running the UI on a workflow and making sure the N=1 window shows as expected.

The current n-window implementation is orthogonal to that old prototype. These Ghosts literally weren't used anywhere, but the args were still in the API and being filtered for.
But sure, test `n=1` if like :)